### DISCUSSION TITLE
Game update broke my Wineskin setup

### DISCUSSION THREAD

**Discussion by TheDarkEnforcer** [The-Wineskin-Project/WineskinServer#277] [Q&A] [answered]
Hello,

In March, I used Wineskin Winery to successfully run Ephinea's Phantasy Star Online Blue Burst client on my 2017 Macbook Pro.

Recently, an update was made to the files of the client which caused the launcher, a file named "online.exe" to stop working properly through my Wineskin Winery setup.

I posted in Ephinea's support section, and have been given 2 different pieces of advice, which I ill quote below:

1. "New launcher uses .NET Framework 4.6.2.

So, get your Windows emulator to be compatible with that and you should be good."

2. "I completely wiped my PSO wine prefix and used Winetricks to install dotnet462 and the game now works.

Specifically I ran this command "winetricks --force dotnet462 corefonts"

(my username over there), Wineskin has Winetricks tool built in to it in the advanced menu."

I cannot wrap my head around how to go about either of these 2 processes, and was hoping to solicit support here.

I do not know how I'd be able to make Wineskin or my setup compatible with .NET Framework 4.6.2

I also cannot find winetricks in any menu within the base Wineskin Winery application.

Hopefully someone here can help give me an inkling of how I'd go about completing these 2 different recommended actions.

**Comment by emendelson** [comment, marked as answer]
Show the contents of the Wineskin application. Launch the Wineskin app inside the Wineskin app. From the menu, choose Advanced. Then go to the Tools tab. In the center column, under Utilities, click Winetricks. Under DLLs, add a checkmark to the box next to dotnet462. Under Fonts, add a checkmark next to core fonts. Click Run.
